Limelight/ViewControllers/StreamFrameViewController.m

```diff
@@ -191,12 +191,15 @@ - (void)connectionTerminated:(long)errorCode {
     dispatch_async(dispatch_get_main_queue(), ^{
         // Allow the display to go to sleep now
         [UIApplication sharedApplication].idleTimerDisabled = NO;
-        
-        UIAlertController* conTermAlert = [UIAlertController alertControllerWithTitle:@"Connection Terminated" message:@"The connection was terminated" preferredStyle:UIAlertControllerStyleAlert];
-        [conTermAlert addAction:[UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault handler:^(UIAlertAction* action){
+        if (errorCode == 0) {
```

Comment:
This actually wasn't accurate because moonlight-common-c wasn't propagating the error code correctly. I think it was always 0.

But I liked the idea, so I reverse engineered the server termination message that sends us the termination reason and now pass 0 to this callback if the server termination code is SERVER_TERMINATED_INTENDED.